server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -557,7 +555,10 @@ def parser_descriptivemetadata(self, item, descript_el):
             elements = location_el.findall('Property')
             for element in elements:
                 if element.attrib.get('FormalName', '') == 'Country':
-                    item['extra']['country'] = element.attrib.get('Value')
+                    country = element.attrib.get('Value')
+                    item['extra']['country'] = country
+                    # country keywords is CV
+                    item.setdefault('subject', []).extend(self._get_country(country))
```

Comment:
IMO it's better to load all country keywords once using `def _get_country_keywords` and save them into `self`.
It's not critical, but it's better to avoid hitting DB in the loop.